pkg/resource/endpoint/testdata/endpoint/cr/e_cmd_invalid_before_create.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: Endpoint
+metadata:
+kk  name: unit_testing_endpoint
```

Comment:
typo: kk

are these not getting caught in tests?

Reply:
When using yaml Marshal on the input file, it finds a typo field instead of name, and therefore ignored it, and ended up not generating an error.
This could be fixed by validating input files (possibly with YamlEqual).